### PR TITLE
update navigation

### LINKED ISSUE #6120: update navigation in public pages
as per latest changes in commercial site, the new lineup would be:

- product
- industries
- discover
- learn
- pricing

### PR DESCRIPTION
**Pull request by matallo**
close #6120

I updated both header and footer as per issue description

screen shot 2015-11-13 at 16 49 03

CR @xavijam

### CODE REVIEW

**Comment by matallo** [comment]
retest this, please

@xavijam the rest of sites have already updated the navigation (header and footer), can you review or pass it along? thanks!

File: lib/assets/javascripts/cartodb/explore/entry.js [hunk 1]
```diff
@@ -30,6 +31,10 @@ $(function() {
       el: $('.js-user-resources')
     });
 
+    var userDiscoverView = new UserDiscoverView({
+      el: $('.js-user-discover'),
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/explore/entry.js]: That comma is not necessary

File: lib/assets/javascripts/cartodb/public_dashboard/entry.js [hunk 2]
```diff
@@ -35,6 +36,10 @@ $(function() {
       el: $('.js-user-resources'),
     });
 
+    var userDiscoverView = new UserDiscoverView({
+      el: $('.js-user-discover'),
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/public_dashboard/entry.js]: Same here

File: lib/assets/javascripts/cartodb/user_feed/entry.js [hunk 3]
```diff
@@ -31,6 +32,10 @@ $(function() {
       el: $('.js-user-resources')
     });
 
+    var userDiscoverView = new UserDiscoverView({
+      el: $('.js-user-discover'),
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/user_feed/entry.js]: comma!

File: lib/assets/javascripts/cartodb/public_table.js [hunk 4]
```diff
@@ -38,6 +39,10 @@ $(function() {
       el: $('.js-user-resources'),
     });
 
+    var userDiscoverView = new UserDiscoverView({
+      el: $('.js-user-discover'),
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/public_table.js]: copy and paste I guess...

File: lib/assets/javascripts/cartodb/public_common/user_discover_view.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
+var cdb = require('cartodb.js');
+var DiscoverDropdown = require('./user_discover/dropdown_view');
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/public_common/user_discover_view.js]: You don't need this, right?

File: lib/assets/javascripts/cartodb/data_library/entry.js [hunk 6]
```diff
@@ -30,6 +31,10 @@ $(function() {
       el: $('.js-user-resources'),
     });
 
+    var userDiscoverView = new UserDiscoverView({
+      el: $('.js-user-discover'),
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/data_library/entry.js]: ^^^

**Comment by xavijam** [comment]
Working properly in Safari then :)

**Comment by xavijam** [comment]
Staging?

File: lib/assets/javascripts/cartodb/public_common/user_discover_view.js (hunk 5 above)
Comment (matallo) [inline lib/assets/javascripts/cartodb/public_common/user_discover_view.js]: yes, some lines below https://github.com/CartoDB/cartodb/pull/6131/files#diff-b9dd47c8e77132e9ee3b4e8cc70c8936R19

Comment (xavijam) [inline lib/assets/javascripts/cartodb/public_common/user_discover_view.js]: You are right :dancer:

**Comment by matallo** [comment]
indeed, also applied the changes in the comments

staging is in http://matataded04.cartodb-staging.com/data-library

**Comment by matallo** [comment]
http://matataded04.cartodb-staging.com/data-library

**Comment by xavijam** [comment]
A good PR is a closed one.

**Comment by xavijam** [comment]
For me perfect, just the comments I made and good to go.

**Comment by matallo** [comment]
retest this, please

**Pull request merged**